subsys/bluetooth/controller/ll_sw/ull_adv_types.h

```diff
@@ -101,6 +101,8 @@ struct ll_adv_iso_set {
 	struct ull_hdr        ull;
 	struct lll_adv_iso    lll;
 
+	uint32_t big_ref_point;
```

Comment:
Would it make sense to add the unit of this? If not in the name, then perhaps at least in a comment